controller/participants/participantsController.js

```diff
@@ -56,15 +49,14 @@ router.get("/getFeed", async (req, res, next) => {
 });
 
 router.get("/searchTweets", async (req, res, next) => {
-  /* Check the req, if there are required paramaters missing, send err status */
```

Comment:
למה למחוק? הערה חשובה

Reply:
It was like a TODO comment. This is an obvious comment and not specific/helping at all

Reply:
Leave it as TODO so I would know that this is the part responsible for this stuff

Reply:
Reminder